pages/app/views/refinery/admin/pages/_form_new_page_parts.html.erb

```diff
@@ -10,3 +14,7 @@
              :cancel_button_id => "new_page_part_cancel",
              :hide_delete => true %>
 </div>
+
+<% content_for :after_javascript_libraries do %>
+  <%= javascript_include_tag 'refinery/jquery.seourl' %>
```

Comment:
Sorry i've forgot to remove it since we doesn't use seourl, replaced by speaking-url